examples/teams/team-burnham/index.ts

```diff
@@ -12,10 +12,11 @@ function getUserArns(scope: Construct, key: string): ArnPrincipal[] {
 }
 
 export class TeamBurnham extends ApplicationTeam {
-    constructor(scope: Construct) {
+    constructor(scope: Construct, networkPoliciesDir: string) {
         super({
             name: "burnham",
-            users: getUserArns(scope, "team-burnham.users")
+            users: getUserArns(scope, "team-burnham.users"),
+            networkPoliciesDir: networkPoliciesDir
```

Comment:
Same here: rename from network to something more generic, e.g. teamManifestDir.

Reply:
How is it resolved? The name stayed the same.

Reply:
This has been corrected under the new PR: #185